### ISSUE TITLE
Deletion variant called with little deletion support.

### ISSUE THREAD

**Issue by mchaisso** [PacificBiosciences/pbbioconda#394]
**Operating system**

CentOS

**Package name**
pbsv 2.4.1 (commit v2.4.1-1-g427dd26)

**Describe the bug**
We are evaluating SV calling using lra alignments instead of pbmm2. There are false-positive calls made in regions with no reads showing evidence of an SV.

**Error message**
Paste the error message / stack.

**To Reproduce**
A sample region where there should be no SV called is:
https://www.dropbox.com/s/apeqqaipfwpfzwg/region.fp.bam?dl=0

**Expected behavior**
No deletion.
The following deletion is called:
1       151520518       pbsv.DEL.2007   AAAGGGAGTGATTCTTTGTGTCATACTCCACAAAGGGCCATCTCTGGCCTAGGCAGCACTGCAGAAGGCAGGACCATGTACCTGAGACTGATGAGCTGAGTCGGGTGCGAGCCTCACTCAAGGAGAGGGTAGAGAGGGTAGGTACCCACCTATCCCATCCGGCTCTGTTCTTTGAGTTTATGGGACTTCCTTCATCCTGATCACTTGGGTGCCATTGTTACAAGGGGCTGGCTATTCCTACTTTTCTTTTTTCTTTTTCTTTTCTTTTTTTTTTTTTTTTGAGACGGAGTCTTGCTCTGTCACCAGGCTGGAGTACAGTGGCCCAATCTTGGCTCACTGCAGCCTCCACCTCCCAGGTTCAAGTGATTCTCCTGCCTCAGCCTCCCAAGTAGCTGGAGCTACAGGCGTGCGCCACCACGCCCAGCTAATTTTTGTATTTTTAGTAGAGATGGGGTTTCATCCTGTTGGCCAGGATGATCTCAATCTCTTAACCTCGTGATTCGCCTGCCTTGGCCTCCCAAAGTGCTGGGATTACAGGCTTGAGCCTCCGTGCCCAGCCTCCCATTTTTCAAAGTGGGGAAACTGAAGTGACTGCTCCTCCAGCAGCTTGAGAGGCCATGTACACTTGGGGTGGGAAAGTGGGTGGGATTGAGGGGCAAAGTGAGGGAAGAGTGGGATACAGGGTGAAGATGAATTAAGGTTGCTGGTTTCCAGCTCAGGGGCAATCTCCACTTCCTTTACTTCCTCTTAGAAAGAACCTTACACTTGACTCCTACCCAGTCCTAGTGACCCCTGTAAAACCTGGGGAGTGAGAGCTGGGGGAACGCAGGCTTTTCCCTATTCACATGAGCCTGTGGTGTGAGAACTCTGCACCTCTCTCCTTATTTCCCTGGGGCCTCCCTGTTTATCTCCTTATCTCTGCTGTGGCGTCTTGATCTCTAACCCCTCAGCTATGTGTTTACCCCCTCCCTGCTTTGAGGAGTGAGTCAGGGTTGGAGATTTGTTTGAACAGAGTGGTGAGGCTGCCTATAATTTGGAGGTAGCTGGGCTTCTCCCAAAAATGTTTATGAGGTTAGAGGCTGCTTACCTTGTGATTAGTACTGCTCCTTTTTCCCTAGAGAAGAAGGCTCCACTTCTCATCTTGCCTTTTAGAGTACCCAGTAGAGACCCCCCTGCCGGATATTTTTGGGGTGTTCCTAAAGCTCCCTGGAGCCTGCTCATAGTCCTGAGGAAATGCAAGCCTGCTCTAGAAAACAACTAAGTGCACCAGGCAGAGAGGAGCCCCAGGATTTGCTCCTTGACTCTGGGTTGGGGGTGCAGGCCGAGGTCTGTGGCCTTGCTGACTAAGATGCATTCTCCAGTTCTGGAGGATGAGGTGCCAGAGAGAGCCCTAAGTTTTTCATCAGCTCCATTAGACAATTTACCTCATCCACCCTCTTCCCTTGCAGACTTTCCAGAGCTACTTGGGAAAGTAATGCTGCTTCACCCTTGTGAAGAGATGTATGGTTTGCAAATCACTTTTACATCATTCTTGCACATCTATCTACTCTTCCTAAAATGTCGGTTGTGAAATTATCACCCTCATTTTACAGACAGAAAACTGAGGCCCAGAGAGATTAAGTGATTTGCTCAAGGATTAATGGTCATCAATCAGGACTCAGACATCCTGCCTCTGACTTCAAGTTTAGTACTTTGTTTTGAAACAGAGTCTCGCTCTGTCACCTGGGTTGGAGTGCATGGCACAGTCTTGGCTCACTGCCGCCTTGACCTTATGAGCTCAATTGATCCTCTCACCTCAGCTTCCCTAGTAGCTGGGACTGCAGGTGTGCACCACCACACCAGGCTAATTTTTGTACTTTTTGTAGAGACAGGGTCTCATTATGTTGCCCAGGCTGGTCTCAAACTCCTGGACTCAGGTGATCTGCCCTCCTGGGCCTCCCAAAGTGCTGGGATTACAGGCGTGAGCCAACATGCCTAGCCAAGTTTAGTACTTTTTCTTTCCTTCTCTTTATATGTTTTATTATAGGAGTGTCACAGGGCTCACTGCACCTCTCTCCTTATCTCCCTGGGGCCTCTGCCTCCCAGGTTCAAGCGATTCTCCTGCCTCAGCCTCCCAGGCAACTGGGATTACAGGTGCCCACCACCACACCTGGCCAGTGTTTTGTATTTTTAGTAGAGACGGGGTTTCACCATGTTGGCCAGGCTGGCCTTGAACTCCTGACCTCTGGTGATCCACCTGCTTTGGCCTCCCAAAGTTCTGGGATTATAGGCGTGAGCCACTGTGCCTGGCCCAAATAGTTTTTTCGTTTGTTTGTTTTTTTGAGACAGAGTCTCACTCTGTCACCCAGGCTGGAGTGCAATGGCGTGATCTTGGCTCACTGCAGCCTCCGCCCCTTGGGTTTAAGCGATTCTCCCACCTCAGTTTCCCGAGTAGCTGGGATTACAAGCATGCACCACCAGGCCCAGCTAATTTTTATATTTTCAGTGGAGACGGGGTTTCACCATGTTGACCAGGTTGGTCTCAAACTCCAGACCTCAAGTGATCTGCCTGCCTCGGCCTTCCAAAGTGCTGGGATTACAGGCGTGAGCCACCACACCTGGCCCCAGCTTGCTTTTTCACTTAACACTTATTTTGAGATTTATTCATGTCAATGCATGTTTCATAGTTCATTTATTTTCACAGCTGAATAGTAGTTCATTGTACTAATATGTCAGGGTTTATTCATTCATTTTCTAATTAGCCAGGCACGGTGGTGCACGCCTGTAGTCCAGCTGCTTAGGAAGCTGAGATGAGAGGATCAATTGAGCCTGGGAGGTCAAGGCTGCAGTGAGGCATGATCATGCCACTGCACTCCCGCCTGGGTAACACAGTGAGACCCTGTCTCAAAAATAATATATAATTTAAAAATTTGTTTTCTTGTCAGTGGGCCATCTGGGTGGTTTCTGGTATTTTGCCATTGCATGCAATGCTACTGTGAACATATTTCTACATGTCTTATCTTTATATTTCTATAAGTGTAATGAAATCCCCCCAAAATAGCTCATTAGCATTTTAATTTGCCTTTTCCTTTACTGGTGAGGTTGAGCATCTTTCCTATTCCATTAGCCACTTGTTATTTCTTTTTAGAGTTGCCTTCTTAGTCTTTGCTCAGTTTTTCCTTGACCTTCCTCTGCTACTTTTCTGTGTCTGCTTTTTCTAGTATTCCATACTGCTTCTCTAAACTGAAGAGCTGGAGTTTCTCAAAGGGAATTGGGAAAATCCCTGAACATATCACCTGGAACTCATGCAACAACATATTTTTTTGAGAACCTATTCTAGGCCAGCCATTGTACTAAATAAGCTATCCATGTCTTCAAGGAGCTTATGATTTTCTCGGGGGAGATGAATGAATTGTTACAAAAGAATGGGCTCCTTGTTAAAGTGCAGGATATGTAGGTTGCAGTAGGTGCTCTGACTGAGGGAGGAGTGCTGGGATAAGGGTCGGACCCTCAGAGAGGAGATACCTTGAAGGCTGAGCAGGAATTCTCCAAGGAGGCAAGGGAGGAGAGGATACCCCGGTCGGAGAAAGCAACATGCGCACCTCTGGGGAGGTGGGAGCACGTATTGCACTGGAAGACCCGTTAGCAGAAACAGACTTGGCTTTGGATGCCTTCTACAATTGGATCAGGAAAAAGTTGGGGATAGAGTGCAGGGTACATTGGGGTACTGGAGGGAGTCAAAACATTTTATGTGGTATAAACTGTGATGGTCCTGATCATAAGGGGACTCTAATACCATAATAAGGAGTTTTTGACTTTTTGTTTTTAAAAACATTTAATAGTAACATATATACAGAAAAATATGTACATCACAAATGTCCAGTCAAGTAATTTTCACAAACTAGACACATTCCTGTGAGCAGCACCCAGATCAAGAAACAAATCATTACGGGCACTGCAGAAGCCCCCTCATGTCCCCTTCTAATTACCACCTACCTCCACCTCAAGGGTAACCACTATCCTGACTTCTAAAGTGCAAATTATATTAGTTTTGCTTATTTTTGTGCTTTTTATATAAGTGGAATCATACAGTATACTTTTGTGTCTGGATTCTTTTGCTTAACATCTTTGTGAGATTTACTGATGTTGCATGTACAGTGCTTCATTCTTTTTCATTGTTGTACAAAGTTCCATTGTATGAATATATTACAATTTATTCTACTCTTGCTGAACATGTGGGTAATTTCTGGTTTGAGGCTATTTTGAAAAATGCTACTGTGAACTTTCTAGTATACATGCTTTTATGGGAAACATATGTATGTGCCAGGGCAAAGGGAAAGCCTCTCCTTCACCCTCGGAAGTTTTGCTGAAAATCACTGACAAAAGAAAATAGGGAAAAAGGCCCGGGCGCGGTGGCTCACGCCTTTAATCCCAGCACTTTGGGAAGCCGAGGCGGGTGGATTACCTGAGGTCAGGAGTTCGAGAGCAGCCTGCCCAACATGGTGAAACCCCATCTCTACTAAAAATACAAAAATTAGCTGGGTGTGGTGGGACACGCCGGTAATCCCAGCTACTTGGGAGGCTGAGGCAGGAGAATTGCTTGAGCCCGGGAGGTGGAGGTTGCAGTGAGCCGAGATCGTGCCACTGCACTCCAGCCTGGCCGACAGAGCGAGAGGGACTCTGTCTCAAAAAAAAAAAAAAAAAAAAAAAAAAGGCATACAGATTTATTTGATCATAGTTTTAGGTGACATAGGAACCTTCTGAATGAAGACCCAAATATACAGAGGAAAACGTCCATTATTTTTAAATTTTCAATTTTTATTATTTTTTAAAAGCTAGCCTTACTGAATCAGAAGCAGTCCATTCTTAGATTTGAAAAAGTATAGGCAGCTGTGTAGAAATATGATTGGACAAAAAGGGTATGATCTAATGCTGAGAGACTAAATGGGGAAACCCAGCAAGGCCTGTCTGTCTAGATTCTTCTTGACCTCTCTAAGCATTCCTTCCAGGTGTGAGACAGGGCCCTCTCTGGAATGGGGGTCTTATGACTTCAGATCAAGCAAGGTGGGTCAGATAATTTTTTTTTTTTTAGATGGAGTCTTGCTCTTGTCCCCAGGCTGGAGTGCAGTGGCACGATCTTGGCTCACAGCAACCTCCACCTCCCAGGTTCAAGCAATTCTCCTGCCTCAGCCTCCTGAGTAGCTGGGACTACAGGCACCTGACACCACGCCTGGCTAATTTTTCTATTTGTAGTAGAGATGGGGTTTCGCCATGTTGGCCAGGCTGGTCTCGAACTCCTGACCTCGTGATCTGCCCACGCCTCCCAAAGTGCTGAGATTACAGGCATGAGCCACTGCACCCGGCCAGATAATTTCTTTATAGCTAGCTTTTAATAGAAAGAGAGGTGGAAAGTCAGAGTAATATTTTTAGGTTTTATGGCTGGCTTTGGGGAAAAGGGGTTCTGGTTTCTATGACCCACCTTGAGAAGAGGGATTCTAGTTTCTATGGCTAGCCCCGGGATGGGGAGGGGAGGAATGAAAGGCCAGAAAAGAGGTCAGGAGAAAGTCAGAGAAAACTTTTTTCTTCCGAGGTCTTCATTTTGGGGTATCATATTCTGAGATCCTACATTTGCATTTCTGTTGGGTAGAGTAACTAGTGGTGGAATTGCTGTGCCTTAGAGTACTTTTGTGTTTATCTTTGGTCCATGCTTCTGAATTTTCCAGCGATTGTGTTGCTGCGCAATCCCAGCATCAGTGTCTCAGAGTGTCGGTTGTTCCACATCTGCTGAACACTTCGTATTTTTTTTTTATCCATTTCATGTTAGTCATTCTGATAGGTATTTTGTGGCATTGTTTTGTGGTTTTACTTTCCAGCTCTTTGATAATGCCTAATGAAGTTGAACCCCTTGTCATATGTTTAATGACTGTTTGGGTATCCTCACAATGATGTGTCTGTTGGCATCTTCTGCCTGTTTTTCTATTGGGCTGCATGTCTGGCTGGCTGTCTCCCTTTCTTTCTTTAAGTAAACATTTTATTTTGAAATAATTTTAAATTTACAGAAAAGTTGCAAAGTAGTACAGAGAACTCTTATATACCCTTCATCCAGTTTTCCCACAATGTTAACATCTTATGTCACCATGGCACGTTTGTCATATAAAGAAATTAATGTTGGTACAATGCTGTTAACAAAGTTGTAGACTATCTGGGTTTCACCACTTTTTCCACTAATATCTTTTTTCTGATCCAGGATTCAGTCCAGGATCCCATGTTAATCATTTTGTTTAGTAGGAATTCTTTATATATTCTGGACATCTTTTGTCAGATATACATATAGTGAATTTCTTTTGCTCTGTGGGTTGCCATGCAGTGTTTTTTTAAATTTTATTTTATTGTTATTGAGACAGAGTCTTGCTCTGTTGCCCAGGTCGGAGTGCAGTGGCACGATCTTGGCTCACTGCAACCTCTGCCTCCCGGGTTCAGGCAATTCTCCCGTCTCAGCCTCCTGAGTAGCTGGGACTACAGGCACGTGCTACCACACCCGGCTAATTTTTGTATTTTTAGTAGAGATGGGGTTTCAACATGTTGGCCAGGCTGGTCTCGAACTCCTGACCTCGTGATCTACCTACCTTGGCCTCCCCAAAGTGCTGGGACTGTGAACCACTGTGCCCGGCCCCCTTTTTTTTTTTTTTTTTTTTTTTTTTGAGACAGAGTCTCACTCCGTTGCCCAGACTGGAGTGCAATAGCACAGTCTTGGCTCACTGCAACCTCTGCCTCCAGAGTTCAAGCGATCCTCCTGCCTCAGCCTCCTGAGTAGCTGGGACTACAGGCATGTGCCACCACTCCTGGGTAATTTTTGTATTTTCAGTAGAGACAGGGTTTCAGCATGTTGGCCAGGCTGGTCTTGAACTCCTGACCTCAGGTGATCCACCCACCTCGGCCTCCCAAAGTGCTGGGGTTACAGGTGTGAGCCACCACACCAGCCCTCTTAGCAGTGTCTTTTGATGAATAAAAGTTCTAAATTTTAAGGCCGGGCATGGTGGCTCACTCCTGTAATCCCAGCACTTTGGGAGGCCGAGGCGGGTGGATCACATGAGGTCAGGAGTTCAAGACCAGCCTGGGCAACATGGCGAAACCATGTCTCTATCAAAAATACAAAAACTTAGCCAGCTATGGTGGTGCACACCTGTAATCCCAGCTACTCAGGAGGCTGAGGTAGGAGGATTGCTTGAACTGGGGAGTCAGAGGGTGTGGTGAGCCAAGATTGTGCCATTGCATTCCAGCCTGGGTAACAGAATGAGACTCCCTCTCAAAAAAAAAAAAAAAATTATAAGGCATACACTAACATTTTAATAGTGGCTGTTTCTGGGTGGCAGGATTATAAATGCTTATTTTCTTCAGTATATTCTTTTTGTCCCTGCAAGTACTTGAAGGAATATTAATTCCTTTTTTTTTTTTTTAACAAAAAAGATAATTATAGGTCGGGCATGGTGGCTCACACCTGTAATCCAAGCACTTTGGGAGGCCGAGGCAGGCAGATCACGAGGTCAGGAGTTTGAGACCAACCTGGCCAACATGGTGAAACCCCTTCTCTACTAAAAATACAAAAATTAGCCGGACGTTGTGGTGGGTGCCTATAGTCCCAGCTACTTGGGAAGCTGAGGCAGGAGAATCGTTTGAACCCAGGAGGCAGAGGTTGCAGTGAGCTGAGATGGTGCCATTGCACTCCAGCCTGGGTGACAAGGTGAGACTCCGTCTCAAAAAAAAAAAAAAAAAAAAGATAATTATAGATTCACATGCAGTTATAAGAAATAATAGAATTCCCCCATATCCTTTATTCCATTTTTTCCAGTGGTAACATCTTGCAAAACTGTGGTACTCTATCACAACCAGGATATTGGCATTGATACAGTCAAGCTATGGAACATTTCCATCACCAAAAGCACCCCTCATGTTGCCCTTTGAGAGCC      A       .       PASS    IMPRECISE;SVTYPE=DEL;SVLEN=-7945;SVANN=TANDEM   GT:AD:DP:SAC    0/1:61,24:85:23,38,16,8

**Comment by zeeev** [comment]
@mchaisso,

I'll take a look.

**Comment by zeeev** [comment]
Hi @mchaisso,

Closing out this issue since we debugged it, and resolved the issue. 

TL;DR : increasing the `--cluster-max-length-perc-diff` can have the downside of clustering small indels into larger FP dels.
